crates/wasi-parallel/benches/cl/nstream_cpp.cl

```diff
@@ -0,0 +1,18 @@
+
+typedef float BufferArrayType;
+
+__kernel void spir_main(__global BufferArrayType *context,
+                        __global BufferArrayType *bufA,
+                        __global BufferArrayType *bufB,
+                        __global BufferArrayType *bufC) {
+  int idx = get_global_id(0);
+  // __local BufferArrayType scalar;
+  // if (idx == 0) {
+  //   scalar = context[1];
+  // }
+  // barrier (CLK_LOCAL_MEM_FENCE);
+  //const BufferArrayType scalar = 3;
```

Comment:
Remove?

Reply:
Oops, I was using those to test reading from a buffer vs reading from local memory vs hardcoding the "context".

Reading from a buffer is the slowest and hardcoding the value is the fastest. And, since nstream is limited by L3 bandwidth, reading from local memory is not that different from reading from a buffer.